.github/workflows/ghpage.yml

```diff
@@ -0,0 +1,38 @@
+name: GitHub Pages
+
+on:
+  push:
+    branches:
+      - mainline  # Set a branch name to trigger deployment
```

Comment:
use `develop` branch for preview

Reply:
done [8d618f9](https://github.com/awslabs/service-workbench-on-aws-cn/pull/21/commits/8d618f96682f3d2f5812f8e51117cd9d5955c90c)